custom/intrahealth/tests/reports/test_fiche_consommation.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+from mock.mock import MagicMock
+
+from custom.intrahealth.reports.fiche_consommation_report import FicheConsommationReport
+from custom.intrahealth.tests.reports.util import ReportTestCase
+from dimagi.utils.dates import DateSpan
+
+
+class TestFicheConsommation(ReportTestCase):
+
+    def test_report_data(self):
+        mock = MagicMock()
+        mock.couch_user = self.web_user
+        mock.GET = {
+            'location_id': 'd1',
+            'startdate': '2017-11-01',
+            'enddate': '2017-11-30'
+        }
+        mock.datespan = DateSpan(datetime(2017, 11, 1), datetime(2017, 11, 30))
```

Comment:
@emord Without this line tests were not deterministic. It always pulls data for last week and that's why initially tests were passing.